Sources/Clappr_iOS/Classes/Base/Player.swift

```diff
@@ -155,16 +155,25 @@ open class Player: BaseObject {
         }
     }
 
-    fileprivate func unbindPlaybackEvents() {
+    private func unbindPlaybackEvents() {
         for id in playbackEventsListenIds {
             stopListening(id)
         }
 
         playbackEventsListenIds.removeAll()
     }
+    
+    public static func register(plugins: [Plugin.Type]) {
+        if(!hasAlreadyRegisteredPlugins) {
+            var builtInPlugins: [Plugin.Type] = [AVFoundationPlayback.self]
+            #if os (iOS)
```

Comment:
Adding a line before this macro helps to see it. What do you guys think?

Reply:
Done!